docs/cybr_conjur_logon.md

```diff
@@ -16,11 +16,12 @@ cybr conjur logon [flags]
 ### Options
 
 ```
-  -a, --account string    Conjur account
-  -b, --base-url string   Conjur appliance URL
-  -h, --help              help for logon
-  -l, --login string      Conjur login name
-      --self-signed       Retrieve and use self-signed certificate when sending requests to the Conjur API
+  -a, --account string      Conjur account
+      --authn-ldap string   Uses provided Service ID to configure LDAP Authentication
```

Comment:
Same here

Reply:
Pending your answer to my question in the previous comment.